cfn/configs/klaxon/dev/us-east-1/config.yml

```diff
@@ -332,7 +332,9 @@ context:
         healthy_threshold_count: 2
         unhealthy_threshold_count: 6
         matcher:
-          HttpCode: "200-399" # accepts success or redirect
+          HttpCode: "200-299"
```

Comment:
💭 Back to the old value

Reply:
yes! the hope would be that this fix means we'd no longer anticipate a 300 code